src/main/kotlin/no/nav/familie/tilbake/behandling/BehandlingService.kt

```diff
@@ -221,6 +224,28 @@ class BehandlingService(private val behandlingRepository: BehandlingRepository,
         behandlingRepository.update(behandling.copy(ansvarligSaksbehandler = ContextService.hentSaksbehandler()))
     }
 
+    @Transactional
+    fun oppdaterFaktainfo(eksternFagsakId: String,
+                          ytelsestype: Ytelsestype,
+                          eksternId: String,
+                          respons: HentFagsystemsbehandlingRespons) {
+        val behandling = behandlingRepository.finnÅpenTilbakekrevingsbehandling(ytelsestype, eksternFagsakId)
+        if (behandling == null) {
+            throw Feil("Det finnes ikke en åpen behandling for eksternFagsakId=$eksternFagsakId,ytelsestype=$ytelsestype")
+        }
```

Comment:
val behandling = behandlingRepository.finnÅpenTilbakekrevingsbehandling(ytelsestype, eksternFagsakId)
        ?:  throw Feil("Det finnes ikke en åpen behandling for eksternFagsakId=$eksternFagsakId,ytelsestype=$ytelsestype")